File: examples/sample.py
from dataclasses import dataclass


@dataclass(frozen=True)
class Theme:
    name: str
    contrast: float

    def is_readable(self, minimum: float = 4.5) -> bool:
        # Keep comments quiet, but readable.
        return self.contrast >= minimum


themes = [Theme("Vesperveil", 14.02)]
for theme in themes:
    print(f"{theme.name}: {theme.is_readable()}")

File: examples/Sample.java
import java.util.List;

public class Sample {
    record Theme(String name, double contrast) {
        boolean isReadable() {
            // Comments are the only italic category.
            return contrast >= 4.5;
        }
    }

    public static void main(String[] args) {
        var themes = List.of(new Theme("Vesperveil", 14.02));
        themes.stream()
            .filter(Theme::isReadable)
            .forEach(theme -> System.out.println(theme.name()));
    }
}
